nanome/_internal/_integration/_serialization/_generate_molecule_image.py

```diff
@@ -0,0 +1,31 @@
+from nanome._internal._util._serializers import _ArraySerializer, _DictionarySerializer, _LongSerializer, _ByteSerializer
+from nanome._internal._structure._serialization import _ResidueSerializer, _AtomSerializer
+from nanome._internal._util._serializers import _TypeSerializer
+
+class _GenerateMoleculeImage(_TypeSerializer):
+    def __init__(self):
+        temp = _ArraySerializer()
+        temp.set_type(_ResidueSerializer())
+        self.res_serializer = _ArraySerializer()
+        self.res_serializer.set_type(temp)
+
+        temp = _ArraySerializer()
```

Comment:
Serialization would be faster with a dedicated inner ImageSerializer leveraging write_byte_array